db/schema.rb

```diff
@@ -71,6 +71,9 @@
     t.boolean "work_history_completed", default: false, null: false
     t.text "work_history_explanation"
     t.boolean "degrees_completed", default: false, null: false
+    t.text "becoming_a_teacher"
```

Comment:
In the medium-to-long term (maybe even in the short-term) we will want to add tailoring to the personal statement section, meaning this data will live on `ApplicationChoice`, not on `ApplicationForm`.

This is just a conversation starter, rather than a strong steer, but is it worth modelling the data this way from the get-go, i.e:

1. These 3 attributes are added to the `application_choices` table
2. The controller could set the attribute on the `ApplicationForm` model but that model then in turn pushes the update onto its `application_choices`
3. Conversely, you could have attribute reader methods on `ApplicationForm` that just take the data from the first application choice...

Reply:
I'd prefer to go with the current implementation in this PR. It will cause a bit more work once we go to personal statement per choice, but it'll be easier to understand the system in the meantime, and it's clearer which bits need to be updated if we make the switch.

Reply:
👍